### ISSUE TITLE
UI: Website is not usable in eww

### ISSUE THREAD

**Issue by idlip** [emacstv/emacstv.github.io#13]
I get blank page when opening with `eww` (emacs web browser)
Even with curl or `monolith` downloading the page does not render in eww.

This might be due to website being in typescript.

It would a good feature if site can be quickly visited via eww.

Although I do appreciate the care and time taken to build UI for site to be viewed in browser as well!

Thank you for the rss feed, I have subscribed to this via feedbase (https://feedbase.org/group/feedbase.video.emacstv/1)

**Comment by sachac** [comment]
Yeah, we could statically generate the HTML and then just use JS for
progressive enhancement. In the meantime, might be fun to use the Org file
for browsing/searching?

**Comment by idlip** [comment]
Yep, org beats everything. Open raw videos.org link in eww, `M-x org-mode`

 We need more emacs-fluencers (hahaa)

**Comment by xenodium** [comment]
From emacs, also consider elfeed (can load https://emacs.tv/videos.rss)

**Comment by idlip** [comment]
> Also consider rss (elfeed can load https://emacs.tv/videos.rss)

I was using elfeed before, now I moved to gnus.
I consume rss feeds by nntp mode using gwene and feedbase service.

I have it subscribed in feedbase here (eg: https://feedbase.org/group/feedbase.video.emacstv/100 - for 100th article)

We have 3 options in emacs: elfeed, newsticker, gnus. Imo elfeed is comfy to use ;)

**Comment by sachac** [comment]
Does it load fine for you now in Eww? Seems to work for me, although of course the YouTube links are not very useful and need to be loaded with mpv instead...

**Comment by idlip** [comment]
> Does it load fine for you now in Eww? Seems to work for me, although of
> course the YouTube links are not very useful and need to be loaded with mpv
> instead...
>

Yes the site loads well on eww now.

I use a bash link handler as secondary-browser-function and also use embark
to open them as needed. So no worries, works like charm : )

Thank you for the changes!

>
